packages/playwright/src/reporters/markdown.ts

```diff
@@ -18,15 +18,14 @@ import fs from 'fs';
 import path from 'path';
 import type { FullResult, TestCase } from '../../types/testReporter';
 import { resolveReporterOutputPath } from '../util';
-import { BaseReporter, formatTestTitle } from './base';
+import { TerminalReporter } from './base';
 
 type MarkdownReporterOptions = {
   configDir: string,
   outputFile?: string;
 };
 
-
-class MarkdownReporter extends BaseReporter {
+class MarkdownReporter extends TerminalReporter {
```

Comment:
This reporter seems to be a non-terminal one. I think it is more like the json/junit.